aws/validators.go

```diff
@@ -2236,6 +2236,31 @@ func validateWorklinkFleetName(v interface{}, k string) (ws []string, errors []e
 	} else if len(value) > 48 {
 		errors = append(errors, fmt.Errorf("%q cannot be longer than 48 characters", k))
 	}
+	return
+}
 
+func validateMediaConnectFlowName(v interface{}, k string) (ws []string, errors []error) {
+	value := v.(string)
+	if !regexp.MustCompile(`^[0-9A-Za-z]+$`).MatchString(value) {
+		errors = append(errors, fmt.Errorf(
+			"only alphanumeric characters are allowed in %q", k))
```

Comment:
MediaConnect flow names allow - (dash), which is disallowed by the regex

Reply:
Thanks. I fixed.